### ISSUE TITLE
Can I use all 4 PS Cameras upside down?

### ISSUE THREAD

**Issue by AlexTheTech** [cboulay/PSMoveService#577]
As the title says it is there any way i can use all of my 4 cameras upside down.
Im asking this because i dont wanna use any tripods and just wanna mount them to my cieling with some tape or somthing. Is it possible?

Cheers Alex

**Comment by G1jsb3rt** [comment]
Yes, you can use them upside down they just need to see the bulb.

**Comment by AlexTheTech** [comment]
wont they flip my movements upside down though?

**Comment by G1jsb3rt** [comment]
No, i had one of my 3 cameras upside down and it worked just fine. So surely it would work with 4 cameras.

**Comment by AlexTheTech** [comment]
yea but thats the issue, you had one of the three not all of them upside down
Il try and see if it works

**Comment by PlutoniumJones** [comment]
Place the cameras in any position you want and the Tracker Pose Calibration(the part with the calibration mat) will determine the orientation of your cameras upside down or not.
The only potential thing to think about in positioning is putting your cameras on their sides as you would then get a larger vertical than horizontal field of view(still not an issue in calibration.)

**Comment by AlexTheTech** [comment]
Oh thats good as thankyou man!
